velox/functions/lib/SimpleComparisonMatcher.h

```diff
@@ -13,13 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#pragma once
 
-#include "velox/functions/prestosql/SimpleComparisonMatcher.h"
+#include "velox/core/Expressions.h"
 #include "velox/vector/ConstantVector.h"
 
-namespace facebook::velox::functions::prestosql {
+namespace facebook::velox::functions::lib {
```

Comment:
Shall we use the same namespace `facebook::velox::functions` as other files under functions/lib?